test/frontend/profile.test.js

```diff
@@ -2,22 +2,74 @@ require("mock-local-storage");
 const assert = require("assert");
 const MatrixProfile = require("../../frontend/profile").default;
 
+function getProfileDataSample() {
+  return {
+    id: '111',
+    name: 'Nome do fulano',
+    imageUrl: 'http://localhost/img.jpg',
+    email: 'Mail@mail.com',
+  };
+}
+
 describe("Basic profile credential Test", () => {
-  it("should return empty of credentials is not present", () => {
+  it("must return empty of credentials is not present", () => {
     const matrixProfile = new MatrixProfile();
     assert.equal(matrixProfile.loadStoredProfile(), null);
   });
 
-  it("should return data of credentials is present", () => {
-    const profileData = {
-      id: "111",
-      name: "Nome do fulano",
-      imageUrl: "http://localhost/img.jpg",
-      email: "Mail@mail.com"
-    };
+  it("must return false if credentials is not present", () => {
+    const matrixProfile = new MatrixProfile();
+    assert.equal(matrixProfile.isProfileStored(), false);
+  });
+
+  it("must return true if credentials is not present", () => {
```

Comment:
I got confused with this description and the current assert, is that correct?

Reply:
yess I'm sorry CTRL+C CTRL+V :) the correct description is : must return true if user credentials is present"

Reply:
Now it makes sense!